test/library/input.spec.js

```diff
@@ -204,21 +204,28 @@ describe('NumberFormat as input', () => {
     simulateKeyInput(wrapper.find('input'), 'Backspace', 0, 4);
     expect(wrapper.state().value).toEqual('+1 (123) 456 7 89 US');
 
-    //when format and number character are delted
-    wrapper.setProps({value: '+1 (999) 999 9 99 US'})
+    //when a format character is replaced with number
+    wrapper.setProps({value: '+1 (777) 777 7 77 US'});
+    simulateKeyInput(wrapper.find('input'), '8', 8, 9);
+    expect(wrapper.state().value).toEqual('+1 (777) 777 7 77 US');
+  });
+
+  it('should update value if group of characters got deleted with format', () => {
```

Comment:
When you pull from main repo. You will find a test, `should not update value if formatting got deleted'` which we can remove now.